pkg/orchestrator/connectors.go

```diff
@@ -163,3 +165,30 @@ func (c *ConnectorOrchestrator) Update(ctx context.Context, id string, config co
 	r.Skip()
 	return conn, nil
 }
+
+func (c *ConnectorOrchestrator) Validate(
+	ctx context.Context,
+	t connector.Type,
+	config connector.Config,
+) error {
+	logger := log.CtxLogger{Logger: zerolog.Nop()}
```

Comment:
The last thing we need to do is add a logger to `orchestrator.base` so that orchestrators can log stuff, then we can also pass that logger to `NewDispenser`.